src/components/Field/SelectField/SelectField.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, ChangeEvent } from 'react';
+import MenuItem from '@mui/material/MenuItem';
+
+import TextField, { TextFieldProps } from '../TextField';
+
+type SelectFieldOptions = {
+  value: string;
+  label: string;
+};
+
+export type SelectFieldProps = {
+  options: SelectFieldOptions[];
+} & TextFieldProps;
+
+const SelectField = ({
+  options,
+  ...otherProps
+}: SelectFieldProps): JSX.Element => {
+  const [selectedValue, setSelectedValue] = useState('');
```

Comment:
@smurmus @brianclee-gh i thought about this more... will we be able to leverage the `selectedValue` value if its encapsulated in the hajimari component (as opposed to an izakaya component)

Reply:
oh hmm ya if we want access to it we will want it to be a level up and pass down ^